custom_components/tuya_local/devices/valehaus_syzn119_tow_w.yaml

```diff
@@ -0,0 +1,84 @@
+name: VALEHAUS Towel Warmer
+products:
+  - id: 1dxb14emvqzxzc5q
+    name: VALEHAUS SYZN119
+
+primary_entity:
+  entity: switch
```

Comment:
This is probably better as a `climate` device, with the temperature, current_temperature and hvac_action integrated and the switch as hvac_mode.  That would leave just the countdown (Timer) entities as secondary entities.

Reply:
there is another PR for this device, I'm deleting it's converter in this PR